### PR TITLE
add CHANGELOG

### PR DESCRIPTION
**Pull request by gy95**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
add changelogs

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1 @@
+CHANGELOG/README.md
```
Comment (RainbowMango) [inline CHANGELOG.md:1]: Why do we need this `soft link`?

**Comment by RainbowMango** [comment]
I find this is useful when we want to know which feature or flag is added in releases.

**Comment by RainbowMango** [comment]
How about moving it to `pkg/docs/changelog/`? Maybe we can publish it on to website someday.

**Comment by gy95** [comment]
> How about moving it to `pkg/docs/changelog/`? Maybe we can publish it on to website someday.

ping @RainbowMango 
move to docs/

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**